### PR TITLE
Homework_3

### PR DESCRIPTION
**Pull request by shotvlad**
DZ

### CODE REVIEW

**Comment by VadzimAlkhimionak** [comment]
Not DZ
use HOMEWORK or HOMETASKS

File: Task_1/Task_1.cpp [hunk 1]
```diff
@@ -1,20 +1,57 @@
...
 {
-    std::cout << "Hello World!\n";
+    int n = 0;
+    double x = 0;
+
+    cout << "Enter number x: ";
+    cin >> x;
+    cout << "Enter degree n: ";
+    cin >> n;
+    cout << endl;
```
Comment (VadzimAlkhimionak) [inline Task_1/Task_1.cpp:18]: Create function

File: Task_2/Task_2.cpp [hunk 2]
```diff
@@ -1,20 +1,46 @@
-﻿// Task_2.cpp : Этот файл содержит функцию "main". Здесь начинается и заканчивается выполнение программы.
-//
+﻿#include <iostream>
 
-#include <iostream>
+using namespace std;
+
+//2. ЕДИНИЦЫ. Найдите все натуральные числа до m, двоичная запись которых состоит из единиц
+
+void BinaryOne(int upTo);
```
Comment (VadzimAlkhimionak) [inline Task_2/Task_2.cpp]: `binaryOne`

File: Task_2/Task_2.cpp [hunk 3]
```diff
@@ -1,20 +1,46 @@
...
+	BinaryOne(m);
+
+	return 0;
 }
 
-// Запуск программы: CTRL+F5 или меню "Отладка" > "Запуск без отладки"
-// Отладка программы: F5 или меню "Отладка" > "Запустить отладку"
+void BinaryOne(int upTo)
+{
+	int numbers = 3, degree = 2;
```
Comment (VadzimAlkhimionak) [inline Task_2/Task_2.cpp:32]: Why 3 and 2?

File: Task_2/Task_2.cpp [hunk 4]
```diff
@@ -1,20 +1,46 @@
...
+	cout << "Natural numbers up to " << upTo << ", whose binary notation consists of ones: 1";
+
+	while (numbers <= upTo)
+	{
+		cout << ", " << numbers;
+
+		numbers += pow(2, degree);
+
+		degree++;
+	}
```
Comment (VadzimAlkhimionak) [inline Task_2/Task_2.cpp]: Two actions in your function

File: Task_3/Task_3.cpp [hunk 5]
```diff
@@ -1,20 +1,58 @@
...
 
-// Советы по началу работы 
-//   1. В окне обозревателя решений можно добавлять файлы и управлять ими.
-//   2. В окне Team Explorer можно подключиться к системе управления версиями.
-//   3. В окне "Выходные данные" можно просматривать выходные данные сборки и другие сообщения.
-//   4. В окне "Список ошибок" можно просматривать ошибки.
-//   5. Последовательно выберите пункты меню "Проект" > "Добавить новый элемент", чтобы создать файлы кода, или "Проект" > "Добавить существующий элемент", чтобы добавить в проект существующие файлы кода.
-//   6. Чтобы снова открыть этот проект позже, выберите пункты меню "Файл" > "Открыть" > "Проект" и выберите SLN-файл.
+    return meaning;
+}
```
Comment (VadzimAlkhimionak) [inline Task_3/Task_3.cpp:58]: Interesting!

**Review by VadzimAlkhimionak — COMMENTED** [review]
Hi. Fix some comments. Well done

File: Task_3/Task_3.cpp (hunk 5 above)
Comment (shotvlad) [reply Task_3/Task_3.cpp:58]: I tried.

File: Task_1/Task_1.cpp (hunk 1 above)
Comment (shotvlad) [reply Task_1/Task_1.cpp:18]: A separate function?

File: Task_2/Task_2.cpp (hunk 2 above)
Comment (shotvlad) [reply Task_2/Task_2.cpp]: Done

File: Task_2/Task_2.cpp (hunk 3 above)
Comment (shotvlad) [reply Task_2/Task_2.cpp:32]: The first number is already written in the row. If there is 1 or 2, then 1 will be written to the list. And so I did only for the sake of beauty.
It also looks better)

File: Task_2/Task_2.cpp (hunk 4 above)
Comment (shotvlad) [reply Task_2/Task_2.cpp]: ?

Comment (shotvlad) [reply Task_2/Task_2.cpp]: -s15xWudjFU

File: Task_1/Task_1.cpp (hunk 1 above)
Comment (VadzimAlkhimionak) [reply Task_1/Task_1.cpp:18]: Of course

File: Task_2/Task_2.cpp (hunk 2 above)
Comment (VadzimAlkhimionak) [reply Task_2/Task_2.cpp]: Where screenshot?)

File: Task_2/Task_2.cpp (hunk 3 above)
Comment (VadzimAlkhimionak) [reply Task_2/Task_2.cpp:32]: Hm.. I guess these numbers should be CONST

File: Task_2/Task_2.cpp (hunk 4 above)
Comment (VadzimAlkhimionak) [reply Task_2/Task_2.cpp]: cout + while = two actions

**Pull request merged**